endpoints/api/organization.py

```diff
@@ -106,10 +106,10 @@ def org_view(o, teams):
         view["tag_expiration_s"] = o.removed_tag_expiration_s
         view["is_free_account"] = o.stripe_id is None
 
-        if features.QUOTA_MANAGEMENT and features.EDIT_QUOTA:
-            quotas = model.namespacequota.get_namespace_quota_list(o.username)
-            view["quotas"] = [quota_view(quota) for quota in quotas] if quotas else []
-            view["quota_report"] = model.namespacequota.get_quota_for_view(o.username)
+    if features.QUOTA_MANAGEMENT and features.EDIT_QUOTA:
```

Comment:
How will this effect permissions? Since we're removing the check if the user is admin we'll most likely want a check if the user is a team member.

Reply:
> How will this effect permissions? Since we're removing the check if the user is admin we'll most likely want a check if the user is a team member.

will update in order for quota to be accessible only users with members or admin permissions.